functions/request-support.ts

```diff
@@ -205,17 +191,16 @@ async function handler(event) {
       };
     }
 
-    const response = await notion.pages.create(body);
+    await notion.pages.create(body);
```

Comment:
Is the response helpful? Are there cases where create can fail/not succeed, but not raise an exception?

Reply:
Not that I know of. I think I originally declared this because I thought I would return something from the response, but it wasn't used and it's not really necessary.